tools/docs/BUILD

```diff
@@ -33,20 +33,12 @@ py_binary(
     ],
 )
 
-envoy_py_binary(
-    name = "tools.docs.sphinx_runner",
-    deps = [
-        "//tools/base:runner",
-        "//tools/base:utils",
-        requirement("colorama"),
-        requirement("Sphinx"),
-        requirement("sphinx-copybutton"),
-        requirement("sphinx-rtd-theme"),
-        requirement("sphinx-tabs"),
-        requirement("sphinxcontrib-httpdomain"),
-        requirement("sphinxcontrib-serializinghtml"),
-        requirement("sphinxext-rediraffe"),
-    ],
+alias(
```

Comment:
i think on these `alias` -> `entry_point` we probs want an upstream comment, as we have added on the lib deps